### PR TITLE
Jupyter!!

### PR DESCRIPTION
**Pull request by eliska-n**
I'm unsure about the nginx locations.

### CODE REVIEW

File: Site/ASAB Maestro/Descriptors/jupyter.yaml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+     Jupyter: 
+        file:
+          {
+              "Tool": {
+                  "image": "media/tools/jupyter.svg",
+                  "name": "Jupyter Notebook",
+                  "url": "{{PUBLIC_URL}}/jupyter"
+              },
+              "Authorization": {
+                  "tenants": "system"
```
Comment (ateska) [inline Site/ASAB Maestro/Descriptors/jupyter.yaml:47]: Remove.

File: Site/ASAB Maestro/Descriptors/jupyter.yaml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    - "{{SITE}}/{{INSTANCE_ID}}:/config"
+
+
+  entrypoint: ["/bin/bash", "-e", "/config/jupyter_start.sh"]
+
+files:
+  - "jupyter_notebook_config.py": |
+      c.ServerApp.token = "{{JUPYTER_TOKEN}}"
+      c.ServerApp.allow_origin = '*'
+      c.ServerApp.port = 8888
```
Comment (ateska) [inline Site/ASAB Maestro/Descriptors/jupyter.yaml:24]: Assign nicer port based on the well-known table.¨

ie 9002

File: Site/ASAB Maestro/Descriptors/zoonavigator.yaml [hunk 3]
```diff
@@ -39,3 +32,63 @@ asab-config:
           }
 
         if_not_exists: true
+
+
+nginx:
```
Comment (ateska) [inline Site/ASAB Maestro/Descriptors/zoonavigator.yaml:37]: Black passenger.

**Review by ateska — CHANGES_REQUESTED** [review]

**Comment by ateska** [comment]
And fix the table at https://ndocs.teskalabs.com/logman.io/reference/network-ports/

File: Site/ASAB Maestro/Descriptors/jupyter.yaml (hunk 2 above)
Comment (byewokko) [reply Site/ASAB Maestro/Descriptors/jupyter.yaml:24]: 8888 is the Jupyter default port though, even wikipedia lists it as an unofficial well-known port, do we really want to overwrite it?

Comment (ateska) [reply Site/ASAB Maestro/Descriptors/jupyter.yaml:24]: meh ...

File: Site/ASAB Maestro/Descriptors/jupyter.yaml (hunk 1 above)
Comment (eliska-n) [reply Site/ASAB Maestro/Descriptors/jupyter.yaml]: done

File: Site/ASAB Maestro/Descriptors/jupyter.yaml (hunk 2 above)
Comment (eliska-n) [reply Site/ASAB Maestro/Descriptors/jupyter.yaml:24]: so, I keep 8888

**Review by ateska — APPROVED** [review]
... raising!

**Pull request merged**